tests/unit/django_spanner/test_compiler.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+from django.test import SimpleTestCase
+from django.core.exceptions import EmptyResultSet
+from django.db.utils import DatabaseError
+from django_spanner.compiler import SQLCompiler
+from django.db.models.query import QuerySet
+from .models import Number
+
+
+class TestUtils(SimpleTestCase):
+    settings_dict = {"dummy_param": "dummy"}
+
+    def _get_target_class(self):
+        from django_spanner.base import DatabaseWrapper
+
+        return DatabaseWrapper
+
+    def _make_one(self, *args, **kwargs):
+        return self._get_target_class()(*args, **kwargs)
```

Comment:
Since every test method uses a connection [`TestCase.setUp`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp) might be a better fit here.

I'm not sure what benefit we get from `_make_one` and `_get_target_class` here.